### ISSUE TITLE
Remove Python section from "other languages" page?

### ISSUE THREAD

**Issue by tpmccallum** [fermyon/developer#475]
Python now has its own menu and page at https://developer.fermyon.com/spin/python-components

Can we remove the https://developer.fermyon.com/spin/other-languages#python altogether?

Does any of that older information want to be brought to the newer `python-components.md` page?

@dicej - happy to receive advice on this one; not urgent at all, just flagging for future task while thinking about it.

**Comment by karthik2804** [comment]
I think `python` in the other languages section references to `wagi`

**Comment by dicej** [comment]
Yeah, I would just remove Python from the "other languages" section.
